### PR TITLE
Improved: Slider: Close area = grey background

### PR DESCRIPTION
**Pull request by math-GH**
When a slider is open, the left area is clickable and closes the slider.

After:
grey transparent to give the focus to the open slider and mark the visible items as "disabled" while the slider is open.
(similar to #4061)
grafik

Changes proposed in this pull request:

- CSS

How to test the feature manually:

1. open config -> extensions
2. edit an extension
3. see the greyed area left of the slider (that is the new "feature")
4. click on it
5. it closes the slider

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

File: p/themes/base-theme/template.rtl.css [hunk 1]
```diff
@@ -1418,13 +1418,28 @@ br {
...
+	top: 0.5rem;
+	right: 0.5rem;
+	filter: grayscale(100%) brightness(2.5);
+}
+
+#close-slider.active:hover img {
+	background: rgba(0, 0, 0, 0.2);
+	border-radius: 50%;
+}
+
```
Comment (Frenzie) [inline p/themes/base-theme/template.rtl.css]: This and the whole X strikes me as odd, making it both disconnected and as if you have to click on that X.

**Review by Frenzie — COMMENTED** [review]
Btw, `#close-slider` and `#slider` would need a `z-index: 1` in this concept or they come up against `#nav_entries`.
Screenshot 2022-03-27 at 09-44-51 (37) Main stream · FreshRSS

File: p/themes/base-theme/template.rtl.css (hunk 1 above)
Comment (math-GH) [reply p/themes/base-theme/template.rtl.css]: I changed it to the default behavior like the other close areas

Comment (math-GH) [reply p/themes/base-theme/template.rtl.css]: And fixed the CSS.

File: p/themes/base-theme/template.rtl.css [hunk 2]
```diff
@@ -1418,13 +1419,25 @@ br {
 }
 
 #close-slider.active {
+	background: rgba(0, 0, 0, 0.2);
+	font-size: 0;
```
Comment (Frenzie) [inline p/themes/base-theme/template.rtl.css:1464]: I'm not sure what the intent here is. Presumably one of display: none, visibility: hidden or visibility: collapse would communicate the intent properly. ^_^

Comment (math-GH) [reply p/themes/base-theme/template.rtl.css:1464]: The reason for `font-size: 0`:

`<a href="#" id="close-slider"<?= $class ?>>❌</a>`

Similar to: 
https://github.com/FreshRSS/FreshRSS/blob/edge/app/layout/aside_feed.phtml#L118

https://github.com/FreshRSS/FreshRSS/blob/edge/p/themes/base-theme/template.css#L1763-L1774

**Review by Frenzie — APPROVED** [review]

File: app/views/configure/queries.phtml [hunk 3]
```diff
@@ -61,9 +61,7 @@
 	</form>
 
 	<?php $class = $this->displaySlider ? ' class="active"' : ''; ?>
-	<a href="#" id="close-slider"<?= $class ?>>
-		<?= _i('close') ?>
-	</a>
+	<a href="#" id="close-slider"<?= $class ?>>❌</a>
```
Comment (Alkarex) [inline app/views/configure/queries.phtml]: Was there a problem with the `_i('close')` approach?

Comment (math-GH) [reply app/views/configure/queries.phtml]: @Frenzie wrote: "This and the whole X strikes me as odd"

And now it is similar to https://github.com/FreshRSS/FreshRSS/blob/edge/app/layout/aside_feed.phtml#L118

**Pull request merged**